tests/unit/models/gpflow/test_sampler.py

```diff
@@ -171,6 +172,50 @@ def _dim_two_gp(mean_shift: tuple[float, float] = (0.0, 0.0)) -> GaussianProcess
     )
 
 
+@pytest.mark.parametrize(
```

Comment:
It might be worth also adding a `@pytest.mark.parametrize("compile", [False, True])` and compiling the sampler with `tf.function` when `compile` is `True`.

Reply:
I've done this now